server/lib/routes/get-metrics-errors.js

```diff
@@ -29,6 +30,15 @@ function setExtraSentryData(data) {
   if (sentryData) {
     sentryData.release = CONTENT_SERVER_VERSION;
 
+    if (sentryData.stacktrace && sentryData.stacktrace.frames) {
```

Comment:
There is a test below that should hit it, otherwise you need to add to `local.json`:

  "sentry": {
    "endpoint": "https://sentry.prod.mozaws.net/api/31/store/",
    "api_key": "8e7aaaa38fdb46e58ec0b3de61373492"
  }

and look at Sentry Content Dev. If you don't have the above in the config then it will skip it.